tests/sample_size/scripts/test_utils.py

```diff
@@ -0,0 +1,269 @@
+import unittest
+from io import StringIO
+from unittest.mock import patch
+
+from numpy.testing import assert_equal
+
+from sample_size.sample_size_calculator import SampleSizeCalculator
+from sample_size.scripts import utils
+
+
+class UtilsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.TEST_STR = "TEST"
+        self.TEST_SAMPLE_SIZE = 2000
+
+    def test_is_float(self):
+        happy_test_str = "0.1"
+        result = utils.is_float(happy_test_str)
+
+        self.assertEqual(result, True)
+
+        sad_test_str = "test"
+        result = utils.is_float(sad_test_str)
+
+        self.assertEqual(result, False)
+
+        blank_test_str = " "
+        result = utils.is_float(blank_test_str)
+
+        self.assertEqual(result, False)
+
+    def test_get_float_float(self):
+        test_input_str = " 0.05"
+        result = utils.get_float(test_input_str, self.TEST_STR)
+
+        self.assertEqual(result, 0.05)
+
+    def test_get_float_error(self):
+        test_input_str = "test"
+
+        with self.assertRaises(Exception) as context:
+            utils.get_float(test_input_str, self.TEST_STR)
+
+        self.assertEqual(
+            str(context.exception),
+            f"Error: Please enter a float for the {self.TEST_STR}.",
+        )
+
+    @patch("sample_size.scripts.utils.get_float")
+    @patch("sample_size.scripts.utils.input")
+    def test_get_alpha(self, mock_input, mock_get_float):
+        test_input_float = 0.01
+        mock_input.return_value = "n"
+        mock_get_float.return_value = test_input_float
+
+        with patch("sys.stdout", new=StringIO()) as fakeOutput:
+            alpha = utils.get_alpha()
+            self.assertEqual(
+                fakeOutput.getvalue().strip(),
+                f"Using alpha ({alpha}) and default power (0.8)...",
+            )
+
+        self.assertEqual(alpha, test_input_float)
+        mock_input.called_once_with(
+            "Enter the alpha between (between 0 and 0.3 inclusively) " "or press Enter to use default alpha=0.05: "
+        )
+        mock_get_float.called_once_with(test_input_float, "alpha")
```

Comment:
```suggestion
        mock_get_float.assert_called_once_with("0.01", "alpha")
```
it is not doing assertion now, also need to fix the values to be "n" (base on your current code) or "0.01" based on my suggestion